variants/sea_picro/pins_arduino.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+// Pin definitions taken from:
+//    https://github.com/joshajohnson/sea-picro/blob/master/documentation/pinout/sea-picro-top-pinout-rory-render.png
+
+// LEDs not pinned out
+#define PIN_LED (31u)
+
+// Serial
+#define PIN_SERIAL1_TX (0u)
+#define PIN_SERIAL1_RX (1u)
+
+// Not pinned out
+#define PIN_SERIAL2_TX (31u)
+#define PIN_SERIAL2_RX (31u)
+
+// SPI
+#define PIN_SPI0_MISO (20u)
+#define PIN_SPI0_MOSI (23u)
+#define PIN_SPI0_SCK (22u)
+#define PIN_SPI0_SS (31u) // not pinned out
+
+// Not pinned out
+#define PIN_SPI1_MISO (31u)
+#define PIN_SPI1_MOSI (31u)
+#define PIN_SPI1_SCK (31u)
+#define PIN_SPI1_SS (31u)
+
+// Wire
+#define PIN_WIRE0_SDA (2u)
+#define PIN_WIRE0_SCL (3u)
```

Comment:
GPIO2 and 3 are **I2C1** pins per [datasheet](https://datasheets.raspberrypi.com/rp2040/rp2040-datasheet.pdf), not I2C0. (Without [special macros](https://github.com/earlephilhower/arduino-pico/blob/f5f7267f4459d83652ffdec8d3acf4fb9fe743b1/variants/adafruit_feather/pins_arduino.h#L32-L39), Wire0 maps to i2c0).

Actual I2C0 pins would e.g. be GPIO 4 and 5, among many others that are pinned out.